src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -840,17 +833,29 @@ def load_arguments(self, _):
             "enable_advanced_network_observability",
             action="store_true",
             is_preview=True,
+            deprecate_info=c.deprecate(
```

Comment:
To me, it seems you've completely removed the implementation of these options. In that case, the options should be removed rather than marked as deprecated. And this is a breaking change. Please update the major version to `10` (the full version would be something like `10.0.0b1`). Don't forget to update VERSION in `src/aks-preview/setup.py`.